packages/canon/src/components/Icon/icons.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// We can add custom icons to the list outside of Remix
+
+import type { IconMap } from './types';
+import {
+  RiHeartFill,
+  RiArrowDownFill,
+  RiCloudFill,
+  RiArrowLeftFill,
+  RiArrowRightFill,
+  RiArrowUpFill,
+  RiDeleteBin6Line,
+  RiAddLine,
+} from '@remixicon/react';
+
+// List of default icons
+export const defaultIcons: IconMap = {
```

Comment:
Bit of a sidenote, but it feels worth exploring whether we could make the majority or all icons available in an efficient way

Reply:
There's 1700 icons in Remix including different variants (Line, Filled, ...). It's just not where we want to go design wise.

Reply:
But I though of a way today to make sure anyone can add their own icon if they want even if it's not part of the initial set. I'll work on that in a separate PR.